From 3e1348dd30f97c51d555f9f523209bf935b287b1 Mon Sep 17 00:00:00 2001
From: "guoyin.chen" <guoyin.chen@freescale.com>
Date: Wed, 25 Dec 2013 10:45:46 +0800
Subject: [PATCH 275/602] ENGR00242365 Update BUILD_ID for kk4.4.2 alpha rc1

Update BUILD_ID as "1.0.0-alpha-rc1"
Update BUILD_NUMBER as "20140110" to align new release date

Signed-off-by: guoyin.chen <guoyin.chen@freescale.com>
---
 arm2_6dq/build_id.mk       | 4 ++--
 arm2_6sl/build_id.mk       | 4 ++--
 evk_6sl/build_id.mk        | 4 ++--
 hdmidongle_6dq/build_id.mk | 4 ++--
 imx5x/build_id.mk          | 4 ++--
 sabreauto_6q/build_id.mk   | 4 ++--
 sabresd_6dq/build_id.mk    | 4 ++--
 7 files changed, 14 insertions(+), 14 deletions(-)

diff --git a/arm2_6dq/build_id.mk b/arm2_6dq/build_id.mk
index fe2c67b..e3213b0 100644
--- a/arm2_6dq/build_id.mk
+++ b/arm2_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/arm2_6sl/build_id.mk b/arm2_6sl/build_id.mk
index fe2c67b..e3213b0 100644
--- a/arm2_6sl/build_id.mk
+++ b/arm2_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/evk_6sl/build_id.mk b/evk_6sl/build_id.mk
index fe2c67b..e3213b0 100755
--- a/evk_6sl/build_id.mk
+++ b/evk_6sl/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/hdmidongle_6dq/build_id.mk b/hdmidongle_6dq/build_id.mk
index fe2c67b..e3213b0 100644
--- a/hdmidongle_6dq/build_id.mk
+++ b/hdmidongle_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/imx5x/build_id.mk b/imx5x/build_id.mk
index fe2c67b..e3213b0 100644
--- a/imx5x/build_id.mk
+++ b/imx5x/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/sabreauto_6q/build_id.mk b/sabreauto_6q/build_id.mk
index fe2c67b..e3213b0 100644
--- a/sabreauto_6q/build_id.mk
+++ b/sabreauto_6q/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
diff --git a/sabresd_6dq/build_id.mk b/sabresd_6dq/build_id.mk
index fe2c67b..e3213b0 100644
--- a/sabresd_6dq/build_id.mk
+++ b/sabresd_6dq/build_id.mk
@@ -18,5 +18,5 @@
 # (like "CRB01").  It must be a single word, and is
 # capitalized by convention.
 
-export BUILD_ID=1.0.0-rc2
-export BUILD_NUMBER=20131108
+export BUILD_ID=1.0.0-alpha-rc1
+export BUILD_NUMBER=20140110
-- 
1.8.0

